jumanji/environments/logic/sudoku/utils_test.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from itertools import product
+
+import chex
+import jax.numpy as jnp
+import pytest
+
+from jumanji.environments.logic.sudoku.constants import (
+    BOARD_WIDTH,
+    INITIAL_BOARD_SAMPLE,
+    SOLVED_BOARD_SAMPLE,
+)
+from jumanji.environments.logic.sudoku.utils import (
+    get_action_mask,
+    is_puzzle_solved,
+    validate_board,
+)
+
+EMPTY_BOARD = jnp.zeros((BOARD_WIDTH, BOARD_WIDTH), int)
```

Comment:
```suggestion
EMPTY_BOARD = jnp.zeros((BOARD_WIDTH, BOARD_WIDTH), dtype=jnp.int32)
```
Or int64, dunno which you prefer